packages/graphql-codegen-cli/src/config.ts

```diff
@@ -124,6 +131,7 @@ export async function loadCodegenConfig({
       '.yaml': customLoader('yaml'),
       '.yml': customLoader('yaml'),
       '.js': customLoader('js'),
+      '.ts': customLoader('ts'),
```

Comment:
should we consider supporting `mts`?

Reply:
Good point, but then, should we support `mjs` files too?

Reply:
Discussed with @dotansimha and @n1ru4l, let's see later if someone is asking for it
cf: https://github.com/dotansimha/graphql-code-generator/issues/8303